sdcm/cluster.py

```diff
@@ -2186,10 +2186,16 @@ def start_scylla(self, verify_up=True, verify_down=False, timeout=300):
     def stop_scylla_server(self, verify_up=False, verify_down=True, timeout=300, ignore_status=False):
         if verify_up:
             self.wait_db_up(timeout=timeout)
-        if not self.is_ubuntu14():
-            self.remoter.sudo('systemctl stop scylla-server.service', timeout=timeout, ignore_status=ignore_status)
-        else:
-            self.remoter.sudo('service scylla-server stop', timeout=timeout, ignore_status=ignore_status)
+        # workaround for issue #7332
+        try:
+            if not self.is_ubuntu14():
+                self.remoter.run('sudo systemctl stop scylla-server.service',
+                                 timeout=timeout, ignore_status=ignore_status)
+            else:
+                self.remoter.run('sudo service scylla-server stop', timeout=timeout, ignore_status=ignore_status)
+        except CommandTimedOut as details:
```

Comment:
I don't think we got there CommandTimedOut.
In my case we say exit code "-1" and a failure, can you please verify?

also, maybe worth retrying the stop, just to be sure it's stopped.

Reply:
I just noticed that we have a verify_down so maybe not needed.

Reply:
The change of the code is too dangerous here, it practically can affect everything.
Let's move the workaround only to the "drain" nemesis.

Reply:
> I don't think we got there CommandTimedOut.
> In my case we say exit code "-1" and a failure, can you please verify?
> 
> also, maybe worth retrying the stop, just to be sure it's stopped.

Yes in your case it failed with -1. from this the ignore_status=True will help.

Reply:
I think @enaydanov hit the same issue in a different nemesis, so maybe this is the right place for the workaround.
But let's make sure it's a bulletproof.